spec/controllers/idv/in_person/public/usps_locations_controller_spec.rb

```diff
@@ -0,0 +1,42 @@
+require 'rails_helper'
+
+RSpec.describe Idv::InPerson::Public::UspsLocationsController do
+  include Rails.application.routes.url_helpers
+
+  describe '#index' do
+    subject(:action) do
+      post :index,
+           params:
+            { address: { address: '87060 Colby Radial, Stephenmouth, OK 73339-7909',
+                         zip_code: '74120',
+                         state: 'WA',
+                         city: 'Lake Dallas',
+                         street_address: '2215 Merrill Wells' } }
```

Comment:
personal style preference, I like the closing braces to line up with the key
```suggestion
      post :index,
           params: {
             address: {
               address: '87060 Colby Radial, Stephenmouth, OK 73339-7909',
               zip_code: '74120',
               state: 'WA',
               city: 'Lake Dallas',
               street_address: '2215 Merrill Wells',
             },
           }
```